docs/v3/hid.yml

```diff
@@ -907,4 +907,18 @@ paths:
         '400':
           description: Bad request. See response body for details.
       security: []
+  '/user/{id}/clients/{client}':
+    delete:
+      tags:
+        - user
+      summary: Revokes one OAuth Client from a user's profile.
+      responses:
```

Comment:
See comment above about `id` and `client` parameters.

Reply:
This is an auto-generated file assembled from the inline code blocks by running `doocker-compose exec api npm run docs`